test/validators/warper/warper-manager-validator.ts

```diff
@@ -15,7 +17,84 @@ export class WarperManagerValidator {
     },
   ): Promise<void> {
     await expect(registrationSuccessTx)
-      .to.emit(testMeta.warperManager, WARPER_MANAGER_EVENTS.WARPER_REGISTERED)
+      .to.emit(testMeta.warperManager, IWARPER_MANAGER_EVENTS.WARPER_REGISTERED)
       .withArgs(universeId, warperAddress, originalCollectionAddress, assetClass);
   }
+
+  public static async validateWarperDeregistration(
+    deRegistrationSuccessTx: ContractTransaction,
+    warperAddress: string,
+    testMeta: {
+      warperManager: IWarperManager;
+    },
+  ): Promise<void> {
+    await expect(deRegistrationSuccessTx)
+      .to.emit(testMeta.warperManager, IWARPER_MANAGER_EVENTS.WARPER_DEREGISTERED)
+      .withArgs(warperAddress);
+  }
+
+  public static async validateWarperPausedProperly(
+    pauseSuccessTx: ContractTransaction,
+    warperAddress: string,
+    testMeta: {
+      warperManager: IWarperManager;
+      reader: SignerWithAddress;
+    },
+  ): Promise<void> {
+    const warperManager = testMeta.warperManager.connect(testMeta.reader);
+
+    expect((await warperManager.warperInfo(warperAddress)).paused).to.eq(true);
+    await expect(pauseSuccessTx).to.emit(warperManager, IWARPER_MANAGER_EVENTS.WARPER_PAUSED).withArgs(warperAddress);
+  }
+
+  public static async validateWarperUnpausedProperly(
+    unpauseSuccessTx: ContractTransaction,
+    warperAddress: string,
+    testMeta: {
+      warperManager: IWarperManager;
+      reader: SignerWithAddress;
+    },
+  ): Promise<void> {
+    const warperManager = testMeta.warperManager.connect(testMeta.reader);
+
+    expect((await warperManager.warperInfo(warperAddress)).paused).to.eq(false);
+    await expect(unpauseSuccessTx)
+      .to.emit(warperManager, IWARPER_MANAGER_EVENTS.WARPER_UNPAUSED)
+      .withArgs(warperAddress);
+  }
+
+  public static async validateWarperControllerSetProperly(
+    warperAddress: string,
+    expectedControllerAddress: string,
+    testMeta: {
+      warperManager: IWarperManager;
+      reader: SignerWithAddress;
+    },
+  ): Promise<void> {
+    const warperManager = testMeta.warperManager.connect(testMeta.reader);
+    expect(await warperManager.warperController(warperAddress)).to.eq(expectedControllerAddress);
+  }
+
+  public static async validateUniverseWarpers(
```

Comment:
This one more relates to how the Tax Terms are registered with WarperWizardV1, so perhaps this one more relates to warper-wizard-v1-validator